File: 2tdspv/aula52/agenda-backend/src/main/java/edu/curso/agenda/model/Contato.java
package edu.curso.agenda.model;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
public class Contato {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    @Column(length=100)
    private String nome;
    @Column(length=100)
    private String email;
    @Column(length=30)
    private String telefone;
    @Column(length = 255)
    private String imagem;
}

File: 3esoa/aula03/agenda-contato-room/app/src/main/java/edu/curso/agendacontato/Contato.kt
package edu.curso.agendacontato

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "Contato")
data class Contato(
    @PrimaryKey(autoGenerate = true)
    var id : Long,
    val nome : String,
    val telefone : String,
    val email : String
)